docs/triage.md

```diff
@@ -33,27 +33,34 @@ the changeset is feasible and to allow the associated CI run for new contributor
 
 - can this be closed outright?
   - e.g. spam/junk
+  - add the `invalid` label
   - close without comment
 - do we not want to do it?
-  - e.g. have already discussed not wanting to do or duplicate issue
+  - e.g. we have already discussed not wanting to do or it's a duplicate issue
+  - add the appropriate label (e.g. `duplicate`)
   - comment and close
-- are we ok with outside contribution for this?
-  - e.g. the task is relatively straightforward, but no people on our team have the bandwidth to take it on at the moment
+- do we want external contribution for this?
+  - e.g. the task is relatively straightforward, but the core team does not have the bandwidth to take it on
   - ensure that the thread contains all the context necessary for someone new to pick this up
-  - add `help wanted` label
+  - add the `help wanted` label
   - consider adding `good first issue` label
+  - comment acknowledging that that we are accepting contributions for this and add any other details you feel may be necessary to complete the work
```

Comment:
This feels unnecessary to me. The `help wanted` label should indicate we are accepting contributions for this and the new requirement that triaged issues contain AC covers the "other details you feel may be necessary to complete the work" bit.

Reply:
Sure, let's remove it.
```suggestion
```